src/components/WhaleSightingViewer/WhaleSightingViewer.scss

```diff
@@ -0,0 +1,54 @@
+@import "../../App.scss";
+
+* {
+    font: $fontType;
+}
+
+h2 {
```

Comment:
As above, I'm wary of rules that target lots of things - this `h2` rule will target _all_ the `h2`s on the whole site! Please limit its scope to only the specific element you want to target, or set the `h2` rule up at the `App.scss` level if you believe it should be global.